packages/nx/src/daemon/server/project-graph-incremental-recomputation.ts

```diff
@@ -104,12 +105,29 @@ function computeWorkspaceConfigHash(projectsConfigurations: any) {
   return new HashingImpl().hashArray([JSON.stringify(projectsConfigurations)]);
 }
 
+/**
+ * Temporary work around to handle nested gitignores. The parcel file watcher doesn't handle them well,
+ * so we need to filter them out here.
+ */
+function filterUpdatedFiles(files: string[]) {
+  try {
+    const quoted = files.map((f) => '"' + f + '"');
+    const ignored = execSync(`git check-ignore ${quoted.join(' ')}`)
```

Comment:
Is this command going to get too long when too many files are updated i.e. when changing branches?